specs/interop/predeploys.md

```diff
@@ -368,6 +379,115 @@ function relayExpire(bytes32 _expiredHash, uint256 _messageSource) external {
 }
 ```
 
+## OptimismSuperchainERC20Factory
+
+| Constant | Value |
+| -------- | ----- |
+| Address  | TBD   |
+
+### OptimismSuperchainERC20
+
+The `OptimismSuperchainERC20Factory` creates ERC20 contracts that compile to the `SuperchainERC20` [standard](token-bridging.md)
+and grant mint-burn rights to the `L2StandardBridge` (`OptimismSuperchainERC20`).
+These ERC20s are called `OptimismSuperchainERC20` and can be converted back and forth with `OptimismMintableERC20` tokens.
+The goal of the `OptimismSuperchainERC20` is to extend functionalities
+of the `OptimismMintableERC20` so that they are interop compatible.
+
+### Overview
+
+Anyone can deploy `OptimismSuperchainERC20` contracts by using the `OptimismSuperchainERC20Factory`.
+
+#### Proxy
+
+The `OptimismSuperchainERC20Factory` MUST be a proxied predeploy.
+It will follow the
+[`Proxy.sol` implementation](https://github.com/ethereum-optimism/optimism/blob/v1.1.4/packages/contracts-bedrock/src/universal/Proxy.sol)
+and `delegatecall()` to the factory implementation address.
+
+#### Beacon Pattern
+
+It MUST deploy `OptimismSuperchainERC20` as
+[BeaconProxies](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/beacon/BeaconProxy.sol),
+as this is the easiest way to upgrade multiple contracts simultaneously.
+Each BeaconProxy will delegatecall to the implementation address provided by the Beacon Contract.
+
+The Beacon Contract will implement the interface defined
+in [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967).
+
+The implementation MUST include an `initialize` function that
+receives `(address _remoteToken, string _name, string _symbol, uint8 _decimals)` and stores these in the BeaconProxy storage.
+
+#### Deployment history
+
+The `L2StandardBridge` will include a `convert()` function that allows anyone to convert
+between any `OptimismMintableERC20` and its corresponding `OptimismSuperchainERC20`.
+For this method to work, the `OptimismSuperchainERC20Factory` MUST include a deployment history.
+
+### Functions
+
+#### `createOptimismSuperchainERC20`
+
+Creates an instance of the `OptimismSuperchainERC20` contract with a set of metadata defined by:
+
+- `_remoteToken`: address of the underlying token in its native chain.
+- `_name`: `OptimismSuperchainERC20` name
+- `_symbol`: `OptimismSuperchainERC20` symbol
+- `_decimals`: `OptimismSuperchainERC20` decimals
+
+```solidity
+createOptimismSuperchainERC20(address _remoteToken, string memory _name, string memory _symbol, uint8 _decimals) returns (address)
+```
+
+It returns the address of the deployed `OptimismSuperchainERC20`.
+
+The function MUST use `CREATE3` to deploy its children.
+This will ensure the same address deployment across different chains,
+which is necessary for the [standard](token-bridging.md) implementation.
+The safest way to use `CREATE3` is through
+[CreateX](https://github.com/pcaversaccio/createx), which will be a preinstall in the OP stack.
```

Comment:
Using language like "will be" results in stale specs, you can assume that its part of the OP Stack already here